aws-lambda/lambda_function.py

```diff
@@ -0,0 +1,74 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
may be let's add a url to the original file in the header

Reply:
done